.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -405,6 +405,13 @@ jobs:
       matrix: ${{ fromJson(needs.generate-test-matrix.outputs.render-matrix) }}
       fail-fast: false
     steps:
+      - name: Chown workspace
+        run: |
+          # Ensure the working directory gets chowned back to the current user
+          docker run --rm -v "$(pwd)/../":/v -w /v "${ALPINE_IMAGE}" chown -R "$(id -u):$(id -g)" .
+      - name: Clean workspace
+        run: |
+          rm -rf "${GITHUB_WORKSPACE:?}/*"
```

Comment:
This wouldn't removes files/folders starting with dot, would it?

Reply:
maybe simply this?
```suggestion
          rm -rf "${GITHUB_WORKSPACE:?}/"
```

Reply:
This will error out though unless we recreate the `GITHUB_WORKSPACE`, this `rm -rf` here is just for safety since the chown above should be the thing that actually fixes permissions on the `checkout` 

I also believe `actions/checkout` does a `rm -rf` of the space as well, hence we received the permission denied error on `actions/checkout`